Evaluate impact of small wins to bottom 10 docs pages (#5811)
mattermost/docs: source/welcome/manage-custom-groups.rst

  :target: https://mattermost.com/deploy
  :alt: Available for Mattermost Self-Hosted deployments.

*Not available in legacy Enterprise Edition E10/E20*

> Custom Groups (Beta) reduce noise and improve focus by notifying the right people in a channel at the right time, while maintaining transparency for all members in that channel. Custom user groups let you notify up to 30 users at a time rather than notifing users individually. 

Reviewer: Is it custom groups or custom user groups?

For example, perhaps you want to @mention a cross-functional team about a bug fixes needed for an upcoming feature release, without notifying everyone else in the channel. Using a custom group notifies the cross-functional team immediately, while keeping important stakeholders in the loop on the status of the feature release.

> Once a custom user group has been created, you can mention that group the same way you @mention another Mattermost member. See the `Mention People in Messages <https://docs.mattermost.com/channels/mention-people.html>`__ documentation for details.

Reviewer: ```suggestion
Once a custom user group has been created, you can mention that group the same way you @mention another Mattermost member. See the `mention people in messages <https://docs.mattermost.com/channels/mention-people.html>`__ documentation for details.
```

.. note::
  
  - System Admins need to enable this feature. See our `Mattermost Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html#custom-user-groups>`__ documentation for details. 

>   - From Mattermost v7.2, users require a system role to create and manage custom user groups in Mattermost Channels. 

Author: ```suggestion
  - From Mattermost v7.2, System Admins can limit who can manage custom user groups through a system admin role, instead of all users being able to manage custom user groups.
```

>   - From Mattermost v7.2, System Admins can limit who can manage custom user groups through a system admin role, instead of all users being able to manage custom user groups.

Author: ```suggestion
  - From Mattermost v7.2, System Admins can limit who can manage custom user groups through a system admin role, instead of all users being able to manage custom user groups. See the `system roles <https://docs.mattermost.com/onboard/system-admin-roles.html?`__ for details.
```
